lib/Rule/Operator/In.php

```diff
@@ -122,4 +125,50 @@ public static function rule_match_user_type( $match, $rule, $options ) {
 
 		return $match;
 	}
+
+	/**
+	 * Filters basic page template rule matches by ACF.
+	 *
+	 * Adds support to the following new operator:
+	 *
+	 * - `IN`: Checks whether the page template is included in a set of possible values.
+	 *
+	 * @since  1.1.0
+	 *
+	 * @param  boolean $match   Whether the page template matches the rule.
+	 * @param  array   $rule    ACF location rule.
+	 * @param  array   $options Options containing the value to match.
+	 *
+	 * @return boolean          Whether the page template matches the rule.
+	 */
+	public function rule_match_page_template( $match, $rule, $options ) {
+
+		$post_type = $options['post_type'];
+		$post_id   = $options['post_id'];
+
+		if ( ! $post_type && ! $post_id ) {
```

Comment:
I don't see the need to check of the post type is present here. The only use for it is bellow to exclude in the case of it being "page". And there you have the check to see if it exists before, to have it set to the type of the post.